core/src/banking_stage/consume_worker.rs

```diff
@@ -472,6 +491,8 @@ struct ConsumeWorkerTimingMetrics {
     record_us: AtomicU64,
     commit_us: AtomicU64,
     find_and_send_votes_us: AtomicU64,
+    successful_bank_waiting_us: AtomicU64,
+    failed_bank_waiting_us: AtomicU64,
```

Comment:
nit: What do you think about renaming these to put the success/failure at the end? This would make it so these two fields are next to each other in the alphabetically sorted list on metrics. Ie, something like this:
```
wait_for_bank_success_us
wait_for_bank_fail_us
```